### PR TITLE
Buf fix: exception on 'yfinance.Ticker(..).info' when no holders data…

### PR DESCRIPTION
**Pull request by artkpv**
Fixing exception raised when no holders available. Example tickets are VZ, BK, ACN. 
```
import yfinance
yfinance.Ticker('VZ').info  # Exception 
```

### CODE REVIEW

File: yfinance/base.py [hunk 1]
```diff
@@ -282,14 +282,15 @@ def cleanup(data):
...
         holders = _pd.read_html(url)
-        self._major_holders = holders[0]
-        self._institutional_holders = holders[1]
-        if 'Date Reported' in self._institutional_holders:
-            self._institutional_holders['Date Reported'] = _pd.to_datetime(
-                self._institutional_holders['Date Reported'])
-        if '% Out' in self._institutional_holders:
-            self._institutional_holders['% Out'] = self._institutional_holders[
-                '% Out'].str.replace('%', '').astype(float)/100
+        if holders and len(holders) >= 2:
```
Comment (tino-247) [inline yfinance/base.py:285]: _holders_ usually contains the Major Holders data. I suggest to to check len(holders) > 0 first and then len(holders) > 1

Comment (tino-247) [reply yfinance/base.py:285]: https://finance.yahoo.com/quote/WTR/holders actually has no holders.
_holders = _pd.read_html(url)_ will return a _ValueError_ which should be caught too.

Comment (artkpv) [reply yfinance/base.py]: @tino-247 Thanks. Added this

**Comment by bradmetz** [comment]
fixed with merge of PR #459

**Pull request closed without merging**